presto-main/src/main/java/com/facebook/presto/execution/SqlStageExecution.java

```diff
@@ -846,11 +843,12 @@ private boolean addNewExchangesAndBuffers()
     }
 
     @VisibleForTesting
+    @SuppressWarnings("NakedNotify")
```

Comment:
Also, I don't think this commit should not change behavior, just modify the code to match style and rules.  We could do it in another commit, but I am rewriting this whole class.